index.html

```diff
@@ -50,6 +50,212 @@ <h3 class="box3">
+        <div class="project-card">
+            <img src="images/Snapshoot.png" alt="project1">
+        </div>
+        <h1 class=" title">
+            Multi-Post
+            Stories
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
+
+    <section class="projects">
+        <div class="project-card">
+            <img src="images/Snapshoot1.png" alt="project1">
+        </div>
+        <h1 class=" title">
+            Tonic
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
+        </div>
+
+
+    </section>
+
+    <section class="projects">
+        <div class="project-card">
+            <img src="images/Snapshoot3.png" alt="project1">
+        </div>
+        <h1 class=" title">
+            Multi-Post
+            Stories
+        </h1>
+        <div class="role">
+            <div class="role-name">Canopy</div>
+            <div class="role-dot"></div>
+            <div class="role-job">Back End Dev</div>
+            <div class="role-dot"></div>
+            <div class="role-year">2015</div>
+        </div>
+
+        <div class="project-details">
+            A daily selection of privately personalized reads;
+            no accounts or sign-ups required.
+        </div>
+
+        <ul class="skills">
+            <li class="skillset">html</li>
+            <li class="skillset">css</li>
+            <li class="skillset">javaScript</li>
+        </ul>
+
+        <div class="div-button">
+            <button class="button">
+               See Project
+            </button>
```

Comment:
Please Implement the button interactions for all the buttons (enable, hover, etc.) according to the requirement.